pkgs/development/compilers/hare/harec.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, qbe
+, scdoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "harec";
+  version = "0.pre+date=2022-04-25";
```

Comment:
Is this kind of versioning allowed? Wouldn't be better to use the unstable format?

Reply:
https://github.com/NixOS/rfcs/pull/107

Reply:
(OTOH I have not so much hope...)